dired-narrow.el

```diff
@@ -95,25 +95,29 @@ we should act on."
                  (function :tag "Use custom function."))
   :group 'dired-narrow)
 
-(defcustom dired-narrow-exit-when-1-left nil
+(defcustom dired-narrow-exit-when-one-left nil
   "If there is only one file left while narrowing,
-exit minibuffer and call dired-narrow-exit-action."
+exit minibuffer and call `dired-narrow-exit-action'."
+  :type '(boolean)
```

Comment:
They types shouldn't be in a list, at least as far as I'm aware.  But maybe it works the same, supposing the type parser is smart enough.

Still, I would prefer just `'boolean` etc.